src/components/TodoItem/TodoItem.tsx

```diff
@@ -1,17 +1,34 @@
 import React from "react";
 import { ITodo } from "../../models/ITodo";
+import { completeTodo } from "../../store/reducers/UserSlice";
+import { useAppDispatch } from "../../hooks/hooks";
 import s from "./TodoItem.module.scss";
 
-const TodoItem: React.FC<ITodo> = ({ message, date, dateExpiration }) => {
+const TodoItem: React.FC<ITodo> = ({ id, message, date, dateExpiration, completed }) => {
+  const dispatch = useAppDispatch();
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    dispatch(completeTodo(id));
+  };
+
   return (
-    <li className={s.todoItem}>
-      <input type="checkbox" />
+    <li className={`${s.todoItem} ${completed ? s.todoItem_checked : ""}`}>
+      <input
+        className="form-check-input dark"
+        type="checkbox"
+        id="checkboxNoLabel"
+        checked={completed}
+        onChange={handleChange}
+      />
       <div className={s.todoItem_message}>
-        <p>{message}</p>
+        <p className={`${completed ? s.completed : ""}`}>{message}</p>
+      </div>
+      <div className={`${s.todoItem_expData} ${s.todoItem_textColor} ${completed ? s.completed : ""}`}>
+        <p>Creation date</p>
+        <p>{date}</p>
       </div>
-      <p>{date}</p>
-      <div className={s.todoItem_expData}>
-        <p>Expiration data</p>
+      <div className={`${s.todoItem_expData} ${completed ? s.completed : ""}`}>
```

Comment:
the same logic " completed ? s.completed : "" " more than 2 times. Create const *** = completed ? s.completed : ""  before return and put ***  variable in JSX.

Reply:
And the good practice is to not put login into JSX.

Reply:
I moved the class logic to a separate variable before return